flink-streaming-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSink.java

```diff
@@ -150,53 +162,59 @@
 
 	/**
 	 * The default maximum size of part files.
-	 *
-	 * 6 times the default block size
+	 * <p>
+	 * By default, {@code 6 X} the default block size.
```

Comment:
i think we should either not mention "block size" or use a more common term for whatever it stands for.